test/JDBC/expected/temp_table.out

```diff
@@ -63,6 +63,32 @@ int#!#varchar#!#int
 ~~END~~
 
 
+-- BABEL-5273 ALTER COLUMN to another char type
```

Comment:
can we add some tests alter column for other types? with an empty table - like some of the other char types like nchar/nvarchar, money, datetime, int?